### PR TITLE
Portfolio Page Desktop version

### PR DESCRIPTION
**Pull request by henrycode460**
Added media Query for Desktop Version
-Added media query to create a desktop version of the portfolio page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,289 +1,647 @@
...
+          </li>
+          <li class="langIconDetails sec-lang">
+            <img class="iconDetailsImg" src="htmlImage.svg" alt="" />
+            <a class="langIconDetailsLink" href="#"
+              ><h4 class="lang-icon-">HTML</h4></a
+            >
+          </li>
+          <li class="langIconDetails thd-lang">
+            <img class="iconDetailsImg" src="cssImage.svg" alt="" />
+            <a class="langIconDetailsLink" href="#"
```
Comment (AnselemOdims) [inline index.html:570]: - [ ] Please kindly ensure that the language(HTML, CSS, JS)  icons are the same size as the original design. At the moment, yours is looking too big.

### Your Design 
image

### Original Design
image

File: index.html [hunk 2]
```diff
@@ -1,289 +1,647 @@
...
+                    </li>
+                    <li class="tag-btn">
+                      <button type="submit" class="btn-small">css</button>
+                    </li>
+                    <li class="tag-btn">
+                        <button type="submit" class="btn-small">javascript</button>
+                      </li>
+                  </ul>
+
+                  <button type="submit" class="btnNormal">See Project</button>
```
Comment (AnselemOdims) [inline index.html:416]: - [ ] Please ensure that the styling for the button state interactions are implemented according to the design

image

File: index.html [hunk 3]
```diff
@@ -1,289 +1,647 @@
...
+            <a class="navbarTwo-link" href="#portfolio">Portfolio</a>
+          </li>
+          <li class="navbarTwo-list">
+            <a class="navbarTwo-link" href="#about">About</a>
+          </li>
+          <li class="navbarTwo-list">
+            <a class="navbarTwo-link" href="#contact">Contact</a>
+          </li>
+        </ul>
+      </nav>
```
Comment (AnselemOdims) [inline index.html:37]: - [ ] Good job implementing the desktop design, but it seems some of the mobile designs were affected in the process. Please ensure that both mobile and desktop designs align with the original design. In this case, your navbar and headline sections does not match the design.

### Your Design
image

### Original Design
image

File: index.html [hunk 4]
```diff
@@ -1,289 +1,647 @@
...
+          <li class="workInfoDetails first-work">
+            <a class="workInfoDetailsLink firstWorkLink" href="#">CANOPY</a>
+            <img class="counter" src="Counter.svg" alt="" />
+          </li>
+          <li class="workInfoDetails">
+            <a class="workInfoDetailsLink" href="#">Back End Dev</a>
+            <img class="counter" src="Counter.svg" alt="" />
+          </li>
+          <li class="workInfoDetails">
+            <a class="workInfoDetailsLink" href="#">2015</a>
```
Comment (AnselemOdims) [inline index.html:90]: - [ ] Good job implementing the desktop design, but it seems some of the mobile designs were affected in the process. In this case, your project cards does not match the original design. Please ensure that both mobile and desktop designs align with the original design. 

### Your Design
image

### Original Design
image

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @henrycode460 , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: style.css [hunk 5]
```diff
@@ -354,3 +362,222 @@ textarea {
...
+
+  .card-containerTwo {
+    display: none;
+  }
+}
+
+@media (min-width: 768px) {
+  .card-container {
+    display: none;
+  }
```
[the remark this answers is not in the document]
Comment (henrycode460) [reply style.css:386]: @AnselemOdims Kindly observe that the style in the mobile version in figma is not the same as in the mobile. There are different images , different content , different navigation , etc.

[the remark this answers is not in the document]
Comment (henrycode460) [reply style.css:386]: @[](url
Screenshot (78)
Screenshot (79)
)

image different from the one in the mobile version. 
Screenshot (80)

**Review by gtekle — CHANGES_REQUESTED** [review]
Hi @henrycode460 :wave: , 

Good job so far! 
There are some issues that you still need to work on to go to the next project but you are almost there! (Please DM me on Slack before you start working on this changes)


## Highlights

## Required Changes ♻️
_Check the comments under the review._

- [ ] I see that you have duplicated your works section HTML code (one for mobile and one for desktop). You should not duplicate your HTML code for different screen sizes. The only thing you need to add is a media query `css` styling targeting different screen sizes. (`min-width: 768px` for desktop version in this case)

- [ ] In the works section, it was a requirement to use Flexbox to layout the contents of the project card in the previous milestones. However, you manually swapped the image and project details on every other card instead of using `css` Flexbox `order` property. As a result, it is now difficult to make your works section responsive using CSS styling. Please take a look at [this](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Ordering_Flex_Items) tutorial.
works-section-800px


- [ ] You should different background images for the mobile version and desktop version in the headline and footer sections. (e.g. `contact form - background-shape-desktop` and `contact form-shapes-mobile`). This will fix the misalignment you are having the background images.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by henrycode460** [comment]
Hello sir I'm still waiting on you. I have message you on slack but no
response.

**Comment by gtekle** [comment]
Hi @henrycode460 
I am sorry for the inconvenience. May I call you now?

**Comment by henrycode460** [comment]
yes please

**Review by RolandM99 — APPROVED** [review]
<h1>Status: Approve 🥇 ✔</h1>
Hi @henrycode460, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



- [ ] [ **OPTIONAL** ] : You're almost there, your project is complete but, It would be better if you center your project card so that the margin-left and margin-right be the same for all screens in the mobile version. Let's rock and fix this simple issue before starting the new week 😃 
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by henrycode460** [comment]
I am back , my Internet cable shake

**Pull request merged**